federation/v1/status.go

```diff
@@ -0,0 +1,142 @@
+// Copyright Red Hat, Inc.
```

Comment:
better call this file statustype.go?

Reply:
renamed to StatusConditions and statusconditions.go